lib/jekyll/configuration.rb

```diff
@@ -38,6 +38,7 @@ class Configuration < Hash
       'highlighter'   => 'rouge',
       'lsi'           => false,
       'excerpt_separator' => "\n\n",
+      'add_charset'   => true,
```

Comment:
What do you think about making this a sass rule?

```yaml
sass:
  style: compressed
  add_charset: true
```

then we can take care of it over in jekyll-sass-converter. i think this shouldnt' be in the core now that i have had more time to think on it.